app/views/providers/address_selections/show.html.erb

```diff
@@ -39,7 +39,13 @@
         <%= content_tag(:span, @form.errors[:address].first, class: ['govuk-error-message']) %>
       <% end %>
       <% input_error_class = @form.errors[:address].any? ? 'govuk-select--error' : '' %>
-      <%= f.select(:lookup_id, @addresses.collect { |a| [a.full_address, a.lookup_id] }, { include_blank: "#{@addresses.size} addresses found" }, class: "govuk-select govuk-!-width-full #{input_error_class}", id: :address) %>
+      <%= f.select(
+            :lookup_id,
+            @addresses.collect { |a| [a.full_address, a.lookup_id] },
+            { include_blank: "#{@addresses.size} addresses found" },
+            class: "govuk-select govuk-!-width-full #{input_error_class}",
+            id: :address
+          ) %>
```

Comment:
The select code includes the fix: Adding class `govuk-!-width-full`. All that I've changed is to split the select code onto multiple lines so that it's easier to read.